core/kazoo_services/src/kz_services.erl

```diff
@@ -1168,8 +1168,14 @@ get_service_modules() ->
 
 -spec get_filesystem_service_modules() -> atoms().
 get_filesystem_service_modules() ->
+    get_filesystem_service_modules(filelib:wildcard([code:lib_dir('kazoo_services'), "/src/services/*.erl"])).
```

Comment:
@k-anderson yes i know. this is the first approach. if module list  is not available from db, it takes from src/*.erl and if not available take them from default coded below.